dotnet/SK-dotnet.sln

```diff
@@ -52,6 +52,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "nuget", "nuget", "{F4243136
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "KernelBuilder", "..\samples\dotnet\KernelBuilder\KernelBuilder.csproj", "{A52818AC-57FB-495F-818F-9E1E7BC5618C}"
```

Comment:
Not sure where else to put this kind of feedback, but here are some questions I wish it was better at answering.

![image](https://user-images.githubusercontent.com/35783676/225972543-a6fa3e18-b348-44b6-9394-c541b5fcf79b.png)

![image](https://user-images.githubusercontent.com/35783676/225972767-391d21d8-aa7f-40b8-a742-fcc9927c616f.png)
![image](https://user-images.githubusercontent.com/35783676/225973024-e61d8bd1-fd76-469c-ba18-5ed7837e37f5.png)
![image](https://user-images.githubusercontent.com/35783676/225973057-91d64f1c-6e39-4888-bbf9-772516f926ba.png)
![image](https://user-images.githubusercontent.com/35783676/225973138-238a4f93-a809-4b4a-9e54-24369dffad32.png)
![image](https://user-images.githubusercontent.com/35783676/225973310-c1a1f094-3ad6-4b86-b2f2-58a32b3b5e80.png)

Reply:
love these suggestions - by default the sample is only working off the *.md files in the repo which limits some of the usefulness of the Q&A at the moment

Reply:
Ah okay that makes sense. Maybe we could add that detail to the readme or UI to not mislead folks?

Reply:
Initial glance, I read that as 'the filetype used for creating embeddings'. Maybe something more like `Files|Filetypes to memorize.` or similar.

Reply:
How about File Type Filter? 

If we can get this PR in I will make that change, it's quite painful to do so right now as it would require a separate PR to memory-sample (I don't own this PR)